### PR TITLE
Code : issue636 - Fix - refactor HTTP error codes GET /mentorship_relation/{r…

### LINKED ISSUE #636: Fix: refactor HTTP error codes GET /mentorship_relation/{request_id}/task/{task_id}/comments
## Description
As a GSoC student,
I need the correct HTTP error response code to be returned accordingly by Mentorship System API in relation to GET /mentorship_relation/{request_id}/task/{task_id}/comments, 
so that I can represent the error message accordingly to both Mentorship System and Bridge In Tech users.

## Mocks
NA

### Update [Required]
- [ ] 403: Forbidden needs to be returned instead of 400: Bad Request if it has unaccepted state relation

### Enhancement to Update [Optional]
NA
## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by Ritika5901234**
…equest_id}/task/{task_id}/comments

### Description

Fixes #636

### Type of Change:

- Code
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Update Swagger documentation and the exported file at /docs folder

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by vj-codes — CHANGES_REQUESTED** [review]
@Ritika5901234 Any specific reason for changing the f-string format ?I don't think  that was supposed to be changed along with HTTP status.

File: app/api/resources/mentorship_relation.py [hunk 1]
```diff
@@ -821,14 +821,14 @@ class TaskComments(Resource):
...
-            f"{messages.MENTORSHIP_RELATION_DOES_NOT_EXIST}<br>"
-            f"{messages.TASK_DOES_NOT_EXIST}",
+            HTTPStatus.FORBIDDEN: messages.UNACCEPTED_STATE_RELATION["message"],
+            HTTPStatus.UNAUTHORIZED: f"{messages.TOKEN_HAS_EXPIRED['message']}<br>"
+            f"{messages.TOKEN_IS_INVALID['message']}<br>"
+            f"{messages.AUTHORISATION_TOKEN_IS_MISSING['message']}<br>"
+            f"{messages.USER_NOT_INVOLVED_IN_THIS_MENTOR_RELATION['message']}",
+            HTTPStatus.NOT_FOUND: f"{messages.USER_DOES_NOT_EXIST['message']}<br>"
+            f"{messages.MENTORSHIP_RELATION_DOES_NOT_EXIST['message']}<br>"
+            f"{messages.TASK_DOES_NOT_EXIST['message']}"
```
Comment (isabelcosta) [inline app/api/resources/mentorship_relation.py:831]: @Ritika5901234 could you please follow @vj-codes suggestion? The `['message']` would not make the documentation on Swagger UI accurate.

**Review by isabelcosta — CHANGES_REQUESTED** [review]
We just need you to change one little thing, I wrote in my comment :) Other than that, your PR is ready to go :)
Thank you for contributing to this 🎉

**Comment by vj-codes** [comment]
Closing this due to inactivity

**Pull request closed without merging**